common/src/main/java/net/blay09/mods/balm/api/tag/BalmItemTags.java

```diff
@@ -7,37 +7,39 @@
 import net.minecraft.world.item.Item;
 
 public class BalmItemTags {
-    public static final TagKey<Item> BLACK_DYES = TagKey.create(Registries.ITEM, new ResourceLocation("c", "dyes/black"));
-    public static final TagKey<Item> BLUE_DYES = TagKey.create(Registries.ITEM, new ResourceLocation("c", "dyes/blue"));
-    public static final TagKey<Item> BROWN_DYES = TagKey.create(Registries.ITEM, new ResourceLocation("c", "dyes/brown"));
-    public static final TagKey<Item> DIAMONDS = TagKey.create(Registries.ITEM, new ResourceLocation("c", "gems/diamonds"));
     public static final TagKey<Item> COOKING_OIL = TagKey.create(Registries.ITEM, new ResourceLocation("c", "cooking_oil"));
-    public static final TagKey<Item> CYAN_DYES = TagKey.create(Registries.ITEM, new ResourceLocation("c", "dyes/cyan"));
-    public static final TagKey<Item> DYES = TagKey.create(Registries.ITEM, new ResourceLocation("c", "dyes"));
+    public static final TagKey<Item> DIAMONDS = TagKey.create(Registries.ITEM, new ResourceLocation("c", "gems/diamonds"));
```

Comment:
DIAMONDS should come before COOKING_OIL because diamonds are very precious and special.

Reply:
That's not possible, the diamonds would get very oily if you have oil first. 
Imagine: you put your hands on diamonds, then oil. Everything fine. Put your hands on oil, then diamonds -> dirty diamonds. I don't feel good with that.

Reply:
Just to be clear: normal peoples go from bottom to top, that's why this order is correct. I made no mistake in writing my previous comment.

Reply:
I'd have argued against that, but you did post a picture of your dragon, so you *must* be normal.